bigtop-packages/src/common/kibana/do-component-build

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+. `dirname ${0}`/bigtop.bom
+
+
+# Install the specific version of nodejs defined in Kibana
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
+export NVM_DIR="$HOME/.nvm"
+[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+[ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"
+
+# Build issues: shasum command not found in Centos/Fedora
+if [ -f /etc/os-release ]; then
+    . /etc/os-release
+fi
+case ${ID} in
+    centos | fedora)
+        sudo ln -s /usr/bin/sha1sum /usr/bin/shasum
+	;;
+esac
```

Comment:
I tried to run `./gradlew kibana-pkg` after `./gradlew toolchain` on CentOS 7/8 and Fedora 31 with this PR, and came across the following error.

```
+ sudo ln -s /usr/bin/sha1sum /usr/bin/shasum
ln: failed to create symbolic link '/usr/bin/shasum': File exists
```

/usr/bin/shasum seems to be installed via perl-Digest-SHA, so these lines look like unnecessary, at least on x86_64 platform?

```
$ rpm -qf /usr/bin/shasum 
perl-Digest-SHA-6.02-440.fc31.x86_64
```

Reply:
Yep，I added `perl-Digest-SHA` in` bigtop_toolchain/manifests/packages.pp ` to fix issues and add `ln -s /usr/bin/sha1sum /usr/bin/shasum` in `do-component-build` for debugging but forgot to remove it.  
Fixed it in the new commit, thanks.